x-pack/plugins/enterprise_search/public/applications/shared/role_mapping/user_added_info.tsx

```diff
@@ -19,6 +19,8 @@ interface Props {
   roleType: string;
 }
 
+const noItemsPlaceholder = <EuiTextColor color="subdued">&mdash;</EuiTextColor>;
```

Comment:
[optional/personal preference] If this isn't being reused in multiple places, maybe just leave it in inline? 🤷